Runtime-Implementation-Details/Relational-Storage.md

```diff
@@ -2,33 +2,112 @@
 layout: page
 title: Relational Storage
 ---
+Relational storage backend code in Orleans is built on generic ADO.NET functionality and is consequently database vendor agnostic. The Orleans data storage layout is explained in [Runtime Tables](Runtime-Tables). Setting up the connection strings is explained in [Orleans Configuration Guide](http://dotnet.github.io/orleans/Orleans-Configuration-Guide/) and [SQL Tables](http://dotnet.github.io/orleans/Advanced-Concepts/Configuring-SQL-Tables).
```

Comment:
added the info about using the nugget package. about the links, they link to actual pages, but the pages aren't great. I'll handle these at a later time.